<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PauseMenu</class>
 <widget class="QWidget" name="PauseMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToolButton" name="continue_btn">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>530</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/continue.png</normaloff>:/res/continue.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>530</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/save.png</normaloff>:/res/save.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>55</width>
     <height>55</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>600</width>
     <height>350</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>0xProto Nerd Font Mono</family>
     <pointsize>12</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QWidget" name="volumn_bar" native="true">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>540</y>
     <width>450</width>
     <height>40</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="lable_volumn">
      <property name="font">
       <font>
        <family>0xProto Nerd Font Mono</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Volumn: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="horizontalSlider">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
